system-variables.md

```diff
@@ -482,11 +482,20 @@ mysql> SELECT * FROM t1;
 - 作用域：SESSION | GLOBAL
 - 默认值：ON
 - 这个变量用来设置是否开启 `TABLE PARTITION` 特性。目前变量支持以下三种值：
-
     - 默认值 `ON` 表示开启 TiDB 当前已实现了的分区表类型，目前 Range partition、Hash partition 以及 Range column 单列的场景会生效。
     - `AUTO` 目前作用和 `ON` 一样。
     - `OFF` 表示关闭 `TABLE PARTITION` 特性，此时语法还是保持兼容，只是创建的表并不是真正的分区表，而是普通的表。
 
+### `tidb_enable_list_partition`
+
+- 作用域：SESSION
+- 默认值：OFF
+- 这个变量用来设置是否开启 `LIST (COLUMNS) TABLE PARTITION` 特性。
+
+> **警告：**
+>
+> 目前 List partition 和 List COLUMNS partition 为实验特性，不建议在生产环境中使用。
```

Comment:
@crazycs520 Suggest putting the warning at the beginning of the variable. For example:
![image](https://user-images.githubusercontent.com/41534398/111096920-85f79900-857b-11eb-9caf-ed4827ac48e5.png)

Reply:
done.